server/src/main/java/org/tbbtalent/server/api/admin/CandidateAdminApi.java

```diff
@@ -109,42 +109,42 @@ public Map<String, Object> search(@RequestBody SearchCandidateRequest request) {
 
     @PostMapping("findbyemail")
     public Map<String, Object> findByCandidateEmail(@RequestBody CandidateEmailSearchRequest request) {
-        Page<Candidate> candidates = this.candidateService.searchCandidates(request);
+        Page<Candidate> candidates = candidateService.searchCandidates(request);
```

Comment:
This is code written by an Angular programmer! (One of the team who ported the code to Sprint/Angular in 2019!)

In Angular, you always have to specify "this"

Reply:
Yep, thought so. As it was being used in some places and not in others I thought it best to take the opportunity to make it consistent throughout.